src/main/java/utils/ExportResultsUtil.java

```diff
@@ -1,74 +1,55 @@
 package utils;
 
-import refactoring.MoveMethodRefactoring;
-
-import com.intellij.psi.PsiMethod;
-import com.intellij.psi.PsiClass;
+import refactoring.Refactoring;
+import javax.swing.*;
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.nio.file.StandardOpenOption;
 import java.util.List;
-import java.util.function.Function;
-import java.util.Optional;
-
 import static java.nio.file.Files.createFile;
 import static java.nio.file.Files.write;
-import static utils.PsiUtils.getHumanReadableName;
 
 /**
  * Class exports (MoveMethod) refactoring results to the file
  */
 public class ExportResultsUtil {
 
-    //private static final Logger LOGGER = Logging.getLogger(ExportResultsUtil.class);
-
-
     /**
-     * Method exports results of refactoring to the file (Now support only MoveMethodRefactoring)
-     * @param refactorings List with offerings to refactoring
-     * @param directory of the file
+     * Exports refactoring results to the file.
      */
-    public static void exportToFile(List<MoveMethodRefactoring> refactorings, String directory) {
-        exportToFile(refactorings, ExportResultsUtil::defaultRefactoringView, directory);
+    public static void export(List<? extends Refactoring> refactorings) {
+        final JFileChooser fileChooser = new JFileChooser();
```

Comment:
Investigate if it possible to use modal window that is responsible to choose destination of the new file instead of that.